string person_id
float32[3] spineBase
float32[3] spineMid
float32[3] neck
float32[3] head
float32[3] shoulderLeft
float32[3] elbowLeft
float32[3] wristLeft
float32[3] handLeft
float32[3] shoulderRight
float32[3] elbowRight
float32[3] wristRight
float32[3] handRight
float32[3] hipLeft
float32[3] kneeLeft
float32[3] ankleLeft
float32[3] footLeft
float32[3] hipRight
float32[3] kneeRight
float32[3] ankleRight
float32[3] footRight
float32[3] spineShoulder
float32[3] handTipLeft
float32[3] thumbLeft
float32[3] handTipRight
float32[3] thumbRight